Meadowlark-js/packages/meadowlark-core/test/handler/CrudHandler/Create.test.ts

```diff
@@ -402,7 +402,7 @@ describe('when posting a request to create a new resource', () => {
       let mockRequestValidator: any;
       let mockDynamo: any;
       const metaEdHeaders = { header: 'one' };
-      const location = `/v3.3b/ed-fi/academicWeeks/0c48a5757d4a3589eada048f37bcf7cf832a77c1dc838152ff2dadcb`;
+      const location = `/v3.3b/ed-fi/academicWeeks/0e44c7538d2ada64aa2ab80a69994f18e05c9f101df8f6c57689c2d3`;
```

Comment:
The hash changed because the "primary key" construction changed. Did that change also ripple through to DynamoDB so that this test is equally applicable to MongoDB and DynamoDB? If not... let's discuss before implementing. Need to decide for how long we keep DynamoDB around in this way.